pyinfra_docker/docker.py

```diff
@@ -11,7 +11,25 @@
 from pyinfra.operations import apt, dnf, files, yum
 
 
-def _apt_install():
+DEFAULTS = {
+    "docker_version": None,
+}
+
+
+def get_pkgs_to_install():
+    docker_packages = [
+        "docker-ce",
+        "docker-ce-cli",
+        "docker-ce-rootless-extras",
+    ]
+    if not host.data.docker_version:
+        return docker_packages
+
+    return [f"{pkg}={host.data.docker_version}" for pkg in docker_packages]
```

Comment:
This will need to use `-` to join name+version under `yum`/`dnf` (https://unix.stackexchange.com/questions/151689/how-can-i-instruct-yum-to-install-a-specific-version-of-package-x).